# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../qmail/qmail-ldap-use-runit.diff
# Copyright (C) 2008 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff --git a/install.c b/install.c
index 1729fd9..cfdd6a8 100644
--- a/install.c
+++ b/install.c
@@ -210,7 +210,7 @@ int mode;
     strerr_die4sys(111,FATAL,"unable to write .../",subdir,"/run: ");
   if (substdio_puts(&ssout, loguser) == -1)
     strerr_die4sys(111,FATAL,"unable to write .../",subdir,"/run: ");
-  if (substdio_puts(&ssout, " multilog t ") == -1)
+  if (substdio_puts(&ssout, " svlogd -tt ") == -1)
     strerr_die4sys(111,FATAL,"unable to write .../",subdir,"/run: ");
   if (substdio_puts(&ssout, home) == -1)
     strerr_die4sys(111,FATAL,"unable to write .../",subdir,"/run: ");
